datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/glossary/mappers/GlossaryNodeMapper.java

```diff
@@ -54,6 +54,11 @@ private GlossaryNodeProperties mapGlossaryNodeProperties(@Nonnull DataMap dataMa
 
   private void mapGlossaryNodeKey(@Nonnull GlossaryNode glossaryNode, @Nonnull DataMap dataMap) {
     GlossaryNodeKey glossaryNodeKey = new GlossaryNodeKey(dataMap);
+
+    if (glossaryNode.getProperties() == null) {
```

Comment:
I don't think this is the right solve. Are there any required fields inside Properties? If yes then GraphQL will still throw.

Reply:
the only required field is `name` which is being set just below :)

Reply:
i see... another way is to just return and not return properties.. will this hide bad data?

Reply:
it looks like the logic here is to provide the glossaryNodeKey's name as the properties name as a fallback, so this logic is just trying to be consistent w/ that